extensions/wikia/PageHeader/Language.class.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Created by PhpStorm.
```

Comment:
It doesn't give a lot of value, I'd remove it. You can disable generating these comments by removing `#parse("PHP File Header.php")` from PhpStorm / Settings / Editor / File and Code Templates / PHP File.